scripts/rpmdb_dump

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
why do we add those scripts instead of just telling people to use command directly?

also, you better to use `exec /usr/bin/…` :)

Reply:
Because there's 20+ years worth of rpmdb recovery documentation out there that can't be changed?